cssparse/shared/src/main/scala/cssparse/Ast.scala

```diff
@@ -0,0 +1,65 @@
+package cssparse
+
+// According to https://www.w3.org/TR/css-syntax-3/
+
+object Ast {
+
+  abstract class ComponentValue
```

Comment:
All of these should be `sealed` to make sure people don't accidentally inherit from them, and to send a message to future programmers that this is a "sealed trait hierarchy"